src/set_theory/ordinal/basic.lean

```diff
@@ -445,6 +444,9 @@ def well_ordering_rel : σ → σ → Prop := embedding_to_cardinal ⁻¹'o (<)
 instance well_ordering_rel.is_well_order : is_well_order σ well_ordering_rel :=
 (rel_embedding.preimage _ _).is_well_order
 
+instance is_well_order.subtype_nonempty : nonempty {r // is_well_order σ r} :=
+⟨⟨well_ordering_rel, infer_instance⟩⟩
```

Comment:
Is it in the `ordinal` namespace?

Reply:
No, this instance isn't contained in an enclosing namespace. I think that's how it should be.

Reply:
I mean, what is the full name of this instance?

Reply:
`is_well_order.subtype_nonempty` is the full name.

Reply:
You're right. Thanks!
bors merge